tests/test_trackers.py

```diff
@@ -327,7 +327,7 @@ def test_dump_and_restore_as_json(default_agent, tmpdir_factory):
     for tracker in trackers:
         out_path = tmpdir_factory.mktemp("tracker").join("dumped_tracker.json")
 
-        dumped = tracker.current_state(should_include_events=True)
+        dumped = tracker.current_state(EventVerbosity.AFTER_RESTART)
```

Comment:
shall we add three quick tests for the `EventVerbosity`s `NONE`, `ALL` and `APPLIED`?

Reply:
good point :+1: